projects/helper/unwrapLPs.js

```diff
@@ -82,12 +82,18 @@ async function unwrapUniswapLPs(balances, lpPositions, block, chain='ethereum',
         const token0 = (await tokens0).output.find(call=>call.input.target === lpToken).output
         const token1 = (await tokens1).output.find(call=>call.input.target === lpToken).output
         const supply = (await lpSupplies).output.find(call=>call.input.target === lpToken).output
-        const {_reserve0, _reserve1} = (await lpReserves).output.find(call=>call.input.target === lpToken).output
+        // -- Add try/catch block in this section, as sometimes some lp do not have _reserve0, _reserve1 ---
```

Comment:
Which LPs don't have these methods?

Reply:
I think in one of the scripts I run, it had lp's from balance and 1inch, which were giving problems. I think those have not this method. Not in this particular script, but another. But adding that protection there, avoids probably the script to stop in the other scenarios, until another helper is built probably to handle balancer/1inch pools data

Reply:
It might be better to leave as it is. The problem is that sometimes RPC calls error out not because there's a problem with the call but because the service is down or it fails that call. If we allow failures on things that should be working then the adapter will ignore the failed calls and compute a TVL that is wrong (this has happened multiple times already)